kore/src/SMT.hs

```diff
@@ -435,6 +445,8 @@ data Config =
         -- ^ optional log file name
         , timeOut :: TimeOut
         -- ^ query time limit
+        , resetInterval :: ResetInterval
```

Comment:
Why aren't the fields in `Config` strict?

Reply:
This was probably a mistake.